examples/multimodal/multimodal_llm/neva/neva_pretrain_energon.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
plz hide this example for now, nemo run team only wants to display nemo-run examples, unless the script is used for tests

Reply:
done